[new-parser] Broken cron and interval timer attributes

## Parent issue
- #5678 

## Failing tests
- `org.drools.model.codegen.execmodel.CepTest#testCalendarsWithCronAndStartAndEnd`
- `org.drools.model.codegen.execmodel.CepTest#testIntervalTimer`

## Notes
- `PATTERN_DSL` only.

## Rule code snippet
```
rule xxx
  timer (cron: 0 0 0 * * ?) 
```

## Error output
```
### parse : ANTLR4_PARSER_ENABLED = true
14:50:02.849 [main] ERROR o.d.c.k.b.impl.AbstractKieProject.buildKnowledgePackages:280 - Unable to build KieBaseModel:kb
InvalidExpressionErrorResult: (cron: 0 0 0 * * ?)


java.lang.AssertionError: [Message [id=1, kieBase=kb, level=ERROR, path=r0.drl, line=-1, column=0
   text=(cron: 0 0 0 * * ?)]]

	at org.drools.model.codegen.execmodel.BaseModelTest.createKieBuilder(BaseModelTest.java:157)
	at org.drools.model.codegen.execmodel.BaseModelTest.createKieBuilder(BaseModelTest.java:132)
	at org.drools.model.codegen.execmodel.BaseModelTest.getKieContainer(BaseModelTest.java:121)
	at org.drools.model.codegen.execmodel.BaseModelTest.getKieContainer(BaseModelTest.java:114)
	at org.drools.model.codegen.execmodel.BaseModelTest.getKieSession(BaseModelTest.java:110)
	at org.drools.model.codegen.execmodel.CepTest.testCalendarsWithCronAndStartAndEnd(CepTest.java:1223)
```

--- a/drools-drl/drools-drl-parser-tests/src/test/java/org/drools/drl/parser/antlr4/MiscDRLParserTest.java
+++ b/drools-drl/drools-drl-parser-tests/src/test/java/org/drools/drl/parser/antlr4/MiscDRLParserTest.java
@@ -1907,6 +1907,24 @@ class MiscDRLParserTest {
         assertThat(at.getValue()).isEqualTo("true");
     }
 
+    @Test
+    public void parse_Timer() throws Exception {
+        final RuleDescr rule = parseAndGetFirstRuleDescrFromFile("rule_timer_attribute.drl" );
+        assertThat(rule.getName()).isEqualTo("simple_rule");
+        assertThat((String) rule.getConsequence()).isEqualToIgnoringWhitespace( "bar();");
+
+        final Map<String, AttributeDescr> attrs = rule.getAttributes();
+        assertThat(attrs.size()).isEqualTo(2);
+
+        AttributeDescr at = (AttributeDescr) attrs.get( "timer" );
+        assertThat(at.getName()).isEqualTo("timer");
+        assertThat(at.getValue()).isEqualTo("int: 0 1; start=1_000_000, repeat-limit=0");
+
+        at = (AttributeDescr) attrs.get( "lock-on-active" );
+        assertThat(at.getName()).isEqualTo("lock-on-active");
+        assertThat(at.getValue()).isEqualTo("true");
+    }
+
     @Disabled("Priority : Low | Not written in docs nor other unit tests. Drop the support?")
     @Test
     public void parse_Attributes_alternateSyntax() throws Exception {
@@ -3157,7 +3175,6 @@ class MiscDRLParserTest {
 
     }
 
-    @Disabled("Priority : Low | Implement multi-value annotation. Not written in docs")
     @Test
     public void parse_MultiValueAnnotationsBackwardCompatibility() throws Exception {
         // multiple values with no keys are parsed as a single value

--- a/drools-drl/drools-drl-parser-tests/src/test/resources/org/drools/drl/parser/antlr4/rule_timer_attribute.drl
+++ b/drools-drl/drools-drl-parser-tests/src/test/resources/org/drools/drl/parser/antlr4/rule_timer_attribute.drl
@@ -1,0 +1,26 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+rule simple_rule
+        timer (int: 0 1; start=1_000_000, repeat-limit=0)
+        lock-on-active true
+    when
+        Foo()
+    then
+        bar();
+end

--- a/drools-drl/drools-drl-parser/src/main/antlr4/org/drools/drl/parser/antlr4/DRLParser.g4
+++ b/drools-drl/drools-drl-parser/src/main/antlr4/org/drools/drl/parser/antlr4/DRLParser.g4
@@ -459,7 +459,7 @@ type : (classOrInterfaceType | primitiveType) typeArguments? ( DOT IDENTIFIER ty
 drlArguments : LPAREN drlArgument (COMMA drlArgument)* RPAREN ;
 drlArgument : ( stringId | floatLiteral | BOOL_LITERAL | NULL_LITERAL ) ;
 
-drlAnnotation : AT name=drlQualifiedName ((LPAREN ( drlElementValuePairs | drlElementValue )? RPAREN) | chunk)? ;
+drlAnnotation : AT name=drlQualifiedName (LPAREN ( drlElementValuePairs | drlElementValue | chunk )? RPAREN)? ;
 
 drlElementValuePairs : drlElementValuePair (COMMA drlElementValuePair)* ;
 drlElementValuePair : key=drlIdentifier ASSIGN value=drlElementValue ;
@@ -474,11 +474,11 @@ attribute : name=( 'salience' | 'enabled' ) conditionalOrExpression #expressionA
           | name=( 'no-loop' | 'auto-focus' | 'lock-on-active' | 'refract' | 'direct' ) BOOL_LITERAL? #booleanAttribute
           | name=( 'agenda-group' | 'activation-group' | 'ruleflow-group' | 'date-effective' | 'date-expires' | 'dialect' ) DRL_STRING_LITERAL #stringAttribute
           | name='calendars' DRL_STRING_LITERAL ( COMMA DRL_STRING_LITERAL )* #stringListAttribute
-          | name='timer' ( DECIMAL_LITERAL | chunk ) #intOrChunkAttribute
+          | name='timer' ( DECIMAL_LITERAL | LPAREN chunk RPAREN ) #intOrChunkAttribute
           | name='duration' ( DECIMAL_LITERAL | TIME_INTERVAL | LPAREN TIME_INTERVAL RPAREN ) #durationAttribute
           ;
 
-chunk : LPAREN .+? RPAREN;
+chunk : .+?;
 
 assignmentOperator : ASSIGN
                    |   ADD_ASSIGN

--- a/drools-drl/drools-drl-parser/src/main/java/org/drools/drl/parser/antlr4/DRLVisitorImpl.java
+++ b/drools-drl/drools-drl-parser/src/main/java/org/drools/drl/parser/antlr4/DRLVisitorImpl.java
@@ -64,7 +64,6 @@ import static org.drools.drl.parser.antlr4.Antlr4ParserStringUtils.getTokenTextP
 import static org.drools.drl.parser.antlr4.Antlr4ParserStringUtils.trimThen;
 import static org.drools.drl.parser.antlr4.DRLParserHelper.getTextWithoutErrorNode;
 import static org.drools.drl.parser.util.ParserStringUtils.appendPrefix;
-import static org.drools.drl.parser.util.ParserStringUtils.safeStripDelimiters;
 import static org.drools.drl.parser.util.ParserStringUtils.safeStripStringDelimiters;
 import static org.drools.util.StringUtils.unescapeJava;
 
@@ -323,10 +322,11 @@ public class DRLVisitorImpl extends DRLParserBaseVisitor<Object> {
         AnnotationDescr annotationDescr = new AnnotationDescr(ctx.name.getText());
         if (ctx.drlElementValue() != null) {
             annotationDescr.setValue(getTextPreservingWhitespace(ctx.drlElementValue())); // single value
-        } else if (ctx.chunk() != null) {
-            annotationDescr.setValue(safeStripDelimiters(getTextPreservingWhitespace(ctx.chunk()), "(", ")"));
         } else if (ctx.drlElementValuePairs() != null) {
             visitDrlElementValuePairs(ctx.drlElementValuePairs(), annotationDescr); // multiple values
+        } else if (ctx.chunk() != null) {
+            // A chunk that is neither a single value nor a list of key-value pairs. For example `!*, age` in `@watch(!*, age)`.
+            annotationDescr.setValue(getTextPreservingWhitespace(ctx.chunk()));
         }
         return annotationDescr;
     }

